project_task_category/models/project_task_category.py

```diff
@@ -11,4 +11,5 @@ class ProjectTask(models.Model):
     categ_id = fields.Many2one(
         'project.category',
         'Category',
+        domain=[('code', '=', 'categ_id')]
```

Comment:
@victormartinelicocorp I slept on this, and I think that adding a flag to the Category model can do the trick. It does have have a few details to get right that I can share with you. Do you prefer to go this way?